UX-Guide-Metadata/draft/techniques/epub-metadata/index.html

```diff
@@ -70,23 +70,23 @@ <h2>EPUB Accessibility Metadata</h2>
  			    This document provides techniques for meeting the guidelines of the <a href="https://www.w3.org/2021/09/UX-Guide-metadata-2.0/principles/">User Experience Guide for Displaying Accessibility Metadata</a>. It provides practical examples for extracting information from the ONIX metadata for showing it to the end users.
 			 </aside>
                     
-            <section id="schema-examples">
-                <h3>Schema.org Examples</h3>
-				<section id="schema-org-describing-an-epub">
-				    <h4>Schema.org metadata describing an EPUB</h4>
-				    <p>Here is an example of embedded Schema.org embedded metadata within the OPF file, 
+            <section id="epub-a11y-metadata-examples">
+                <h3>EPUB Accessibility Metadata Examples</h3>
+				<section id="epub-a11y-metadata-describing-an-epub">
+				    <h4>EPUB accessibility metadata describing an EPUB</h4>
+				    <p>Here is an example of accessibility metadata embedded within the OPF file, 
                         which will be used as a reference point for the
                         following examples on EPUB accessibility metadata: the results of the XPath shown are based on
 				        this example.
                     </p>
-				    <aside class="example" id="schema-org-example-1">
+				    <aside class="example" id="epub-a11y-metadata-example-1">
                         <pre><code class="xpath">TBD</code></pre>
                     </aside>
 				</section>
 				<section>
-				    <h4 id="schema-org-describing-an-full-audio-book">Schema.org metadata describing an audiobook</h4>
-				        <p>Here is an example of an Schema.org OPF Metadata for describing an audiobook, which will be used as a reference point for the some of the following examples on EPUB accessibility metadata: the results of the XPath shown are based on this example.</p>
-				        <aside class="example" id="schema-org-example-2">
+				    <h4 id="epub-a11y-metadata-describing-an-full-audio-book">EPUB accessibility metadata describing an audiobook</h4>
+				        <p>Here is an example of an accessibility OPF Metadata for describing an audiobook, which will be used as a reference point for the some of the following examples on EPUB accessibility metadata: the results of the XPath shown are based on this example.</p>
```

Comment:
... for some of... (There's an additional "the" to suppress)

Reply:
Fixed thanks Gautier